### ISSUE TITLE
Zenroom python binding crashes when sass libraries are installed

### ISSUE THREAD

**Issue by sbocconi** [dyne/Zenroom#203]
zenroom seems to load the wrong .so library when particular libraries are installed.

To reproduce, on Ubuntu
```
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS
Release:	20.04
```
execute
```
conda create -n Crash python==3.8
conda activate Crash
pip install -y rcssmin libsass rjsmin zenroom==2.0.0.dev1622374816
python verify_signed_vc.py 
```
where verify_signed_vc.py is [here](https://github.com/LedgerProject/GeneCoop/blob/master/Demonstrator/zencoop/crashes/verify_signed_vc.py)

Errors are of the kind:
`Exception in zenroom call: /home/stefano/miniconda3/envs/Crash/lib/python3.8/site-packages/_rjsmin.cpython-38-x86_64-linux-gnu.so: undefined symbol: zencode_exec_tobuf`

After removing (rm) each library that gives an error the program works fine.

It seems that Zenroom loads the wrong .so libraries.

**Comment by andrea-dintino** [comment]
It seems that this file [https://github.com/dyne/Zenroom/blob/master/bindings/python3/zenroom/_config.py](https://github.com/dyne/Zenroom/blob/master/bindings/python3/zenroom/_config.py) tries to find where Zenroom is and loads instead the file `_rjsmin.cpython-38-x86_64-linux-gnu.so` in the same folder, which looks like this: 

[https://gyazo.com/0bf71160c3744244e5ddbb2da41335f5 ](https://gyazo.com/0bf71160c3744244e5ddbb2da41335f5 )

**Comment by sbocconi** [comment]
So we have seen that in [_config.py](https://github.com/dyne/Zenroom/blob/master/bindings/python3/zenroom/_config.py) the line
```
LIBZENROOM_LOC = os.path.join(parent_dir, fnmatch.filter(
                                          os.listdir(parent_dir), "*.so")[0])
```
first select **all** `*.so` in the directory and then it takes the first one of the list.

If zenroom will always part of the name of the library and the only library containing that string, one possible solution could be to change the filter to:
```
fnmatch.filter(os.listdir(parent_dir), "*zenroom*.so")[0]
```
The match is case insensitive.

**Comment by jaromil** [comment]
solved in #205 many thanks!
